hoot-core/src/main/cpp/hoot/core/cmd/VersionCmd.cpp

```diff
@@ -99,7 +105,8 @@ class VersionCmd : public BaseCommand
 #   ifdef HOOT_HAVE_LIBCPPUNIT
       LOG_DEBUG("CppUnit Version: " << CPPUNIT_VERSION);
 #   endif
-
+    LOG_DEBUG("NodeJs Version: " << NODE_VERSION_STRING);
+    LOG_DEBUG("v8 Version: " << V8_VERSION_STRING);
```

Comment:
At first I thought the JavaScript crashes were somehow being caused by a different NodeJs or v8 version, so I added this version info to the `version` command.  In the end, the version was consistent but I still wanted to leave these in so that we can ensure that they are consistent on our machines, the production machines, and anyone else who uses Hootenanny.